crates/gpui/src/platform/linux/window.rs

```diff
@@ -17,7 +17,10 @@ use std::{
     rc::Rc,
     sync::{self, Arc},
 };
-use xcb::{x, Xid as _};
+use xcb::{
+    x::{self, StackMode},
+    Xid as _,
```

Comment:
Why is this imported as wildcard?

Reply:
multi-line? cargo fmt said so...
StackMode separately from x itself? to access StackMode without the x:: prefix in the function.

In my opinion, it doesn't really matter if I import it separately or use the prefix, 
and (the most important reason) my ide said so. 

Do you think this is bad? I'll open a new pr if you say it's worth it!

Reply:
Sorry, I wasn't clear enough. My question is about `Xid as _` (I don't know how it's called. Placeholder use?) It seems like this import is unused.

Reply:
I found an answer https://doc.rust-lang.org/reference/items/use-declarations.html#underscore-imports